cssFlexbox/css-flexbox_profile/css/styles.css

```diff
@@ -9,20 +9,30 @@ body {
   max-width: 1200px;
 }
 
+.profile {
+  display: flex;
+  flex-direction: column;
+}
 .main__title {
   padding: 10px;
   background: #f3f5f9;
 }
 
 .profile__image {
+  display: flex;
   border-radius: 50%;
   max-width: 50%;
   height: auto;
+  align-self: center;
 }
 
 .tech-stack__list {
   list-style: none;
   padding-left: 0;
+  display: flex;
+  flex-direction: row;
```

Comment:
not sure whether this ` flex-direction: row;` is needed?

Reply:
you are right! thanks